### DISCUSSION TITLE
web版的pag是否有动画完成回调事件

### DISCUSSION THREAD

**Discussion by yz-qc** [Tencent/libpag#434] [Q&A] [answered]
目前在官方例子中没有看到 web版的动画播放完成的回调事件。
在api文档中也未找到相关资料
是否是web版的没有回调事件，或者是在api中藏得太深，我没有找到？

**Comment by zenoslin** [comment, marked as answer]
有动画播放完成的回调事件的，API文档中可以查看在 types.PAGViewListenerEvent 查看所有事件。
示例：
``` javascript
pagView.addListener("onAnimationEnd", () => {})
```

**Comment by wzbj** [comment]
通过事件控制动画重新播放怎么控制？
